frontend/src/views/Collection/components/ActionButtons/components/MoreDropdown/components/Menu/index.tsx

```diff
@@ -0,0 +1,35 @@
+import {
+  IMenuItemProps,
+  Intent,
+  Menu as RawMenu,
+  MenuItem,
+} from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
+import React from "react";
+import DeleteCollection from "src/components/Collections/components/DeleteCollection";
+
+const DeleteButton = (props: IMenuItemProps) => {
```

Comment:
This is the collection level `<MoreDropdown />` component, which has the delete collection button